src/Nancy/GlobalizationConfiguration.cs

```diff
@@ -15,15 +15,22 @@ public class GlobalizationConfiguration
         /// <summary>
         /// Initializes a new instance of the <see cref="GlobalizationConfiguration"/> class
         /// </summary>
-        /// <param name="supportedCultureNames"></param>
-        public GlobalizationConfiguration(IEnumerable<string> supportedCultureNames)
+        /// <param name="supportedCultureNames">An array of supported cultures</param>
+        /// <param name="defaultCulture">The default culture of the application</param>

```

Comment:
You seem to associate the Culture settings only being used with the Culture Conventions? There is nothing stopping someone from grabbing the `GlobalizationConfiguration` and using `SupportedCultureNames` and/or `DefaultCulture` anywhere in their code? It makes sense to make sure the latter is included in the former.